clang/include/clang/CIR/Dialect/Builder/CIRBaseBuilder.h

```diff
@@ -252,11 +252,11 @@ class CIRBaseBuilderTy : public mlir::OpBuilder {
   [[nodiscard]] cir::GlobalOp createGlobal(mlir::ModuleOp mlirModule,
                                            mlir::Location loc,
                                            mlir::StringRef name,
-                                           mlir::Type type,
+                                           mlir::Type type, bool isConstant,
                                            cir::GlobalLinkageKind linkage) {
     mlir::OpBuilder::InsertionGuard guard(*this);
     setInsertionPointToStart(mlirModule.getBody());
-    return create<cir::GlobalOp>(loc, name, type, linkage);
+    return create<cir::GlobalOp>(loc, name, type, isConstant, linkage);
```

Comment:
```suggestion
    return cir::GlobalOp::create(*this, loc, name, type, isConstant, linkage);
```
Since you're modifying this code, let's bring it into conformance with the new idiom.